src/tmpfiles/tmpfiles.c

```diff
@@ -860,15 +860,31 @@ static int path_open_parent_safe(const char *path) {
         return fd;
 }
 
+static int path_open_safe(const char *path) {
+        int fd;
+
+        /* path_open_safe() returns a file descriptor opened with O_PATH after
+         * verifying that the path doesn't contain unsafe transitions, except
+         * for its final component as the function does not follow symlink. */
+
+        assert(path);
+
+        fd = chase_symlinks(path, NULL, CHASE_OPEN|CHASE_SAFE|CHASE_NOFOLLOW, NULL);
+        if (fd == -EPERM)
+                log_error("Unsafe symlinks encountered in %s, aborting.", path);
```

Comment:
errors we recieve should be passed to log_error_errno(), so that they are included in the structure log message even if we don't show them in the text msg